regression-tests.dnsdist/test_Advanced.py

```diff
@@ -1595,3 +1596,25 @@ def testTempFailureTTLBinding(self):
         receivedQuery.id = query.id
         self.assertEquals(query, receivedQuery)
         self.assertEquals(receivedResponse, response)
+
+class TestAdvancedEDNSOptionRule(DNSDistTest):
+
+    _config_template = """
+    newServer{address="127.0.0.1:%s"}
+    addAction(EDNSOptionRule(8), DropAction())
```

Comment:
We might want to export `EDNSOptionCode.Cookie`, `EDNSOptionCode.ECS` and so on to `Lua`.

Reply:
pushed